packages/protocol/tasks/utils.ts

```diff
@@ -70,7 +70,7 @@ function getDeployments(_fileName: string) {
 
 async function getSlot(hre: any, signal: any, mappingSlot: any) {
     return hre.ethers.utils.solidityKeccak256(
-        ["bytes", "uint256"],
```

Comment:
there will be some conflicts with my pending PR: https://github.com/taikoxyz/taiko-mono/pull/6861. I would suggest to change the base of this PR to branch `claimTokens`

Reply:
> there will be some conflicts with my pending PR: #6861. I would suggest to change the base of this PR to branch `claimTokens`

I'm currently refactoring so that we won't need this function! I can change it anyway in the interim though.